tests/Sniffs/WhiteSpace/DocBlock/correct-inside.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="test")
+ * @ORM\InheritanceType("JOINED")
+ * @ORM\DiscriminatorColumn(name="discriminator", type="text")
+ * @ORM\DiscriminatorMap({
+ * 		"complex" = "Test1",
+ * 		"simple" = "Test2"
+ * })
+ */
+class Test
```

Comment:
This test file can be removed, as it's not readable and the feature is covered by `correct-inside2.php` already.